src/einsteinpy/symbolic/einstein.py

```diff
@@ -40,6 +41,10 @@ def __init__(self, arr, syms, config="ll", parent_metric=None):
             arr=arr, syms=syms, config=config, parent_metric=parent_metric
         )
         self._order = 2
+        self._parent_metric = parent_metric
+        self._config = config
+        if isinstance(syms, (list, tuple)):
+            self.syms = syms
```

Comment:
These lines are not required to be added, these are already handled by the parent class :)